auto_remediate/test/test_config_s3.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+import logging
+
+import moto
+import pytest
+
+from .. import config_rules
+
+# s3_bucket_server_side_encryption_enabled
+class TestS3BucketServerSideEncryptionEnabledCheck:
+    @pytest.fixture
+    def cr(self):
+        with moto.mock_s3():
+            cr = config_rules.ConfigRules(logging)
+            yield cr
+
+    # def test_s3_bucket_sse_enabled(self, cr):
```

Comment:
uncomment

Reply:
It's commented out because there's no Moto support :( I implemented the test and then found out it doesn't support those API calls.